### PR TITLE
chore: automate updating formulae

### PR DESCRIPTION
**Pull request by deansheather**
and updates to 1.28.2

I will add a job to the monorepo to automatically run this on release

### CODE REVIEW

File: ci/update.sh [hunk 1]
```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
```
Comment (jawnsy) [inline ci/update.sh:1]: did you run this through shellcheck too? since we don't have that check in this repo

File: ci/update.sh [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+linux_sha="$(echo "$3" | tr "[:upper:]" "[:lower:]")"
+
+# Get the old version to use in our find/replace.
+old_version="$(sed -n "s/.*version \"\(.*\)\".*/\1/p" ../coder-cli.rb)"
+if [[ "$old_version" == "" ]]; then
+  echo "Could not determine the old version of the formula..." >&2
+  exit 1
+fi
+
+# Replace version
```
Comment (jawnsy) [inline ci/update.sh:21]: this does the update but still has to be run manually each release, I guess?

**Review by jawnsy — APPROVED** [review]

File: ci/update.sh (hunk 2 above)
Comment (deansheather) [reply ci/update.sh:21]: I will be adding a job to the monorepo to call this script

File: ci/update.sh (hunk 1 above)
Comment (deansheather) [reply ci/update.sh:1]: yes 👍

**Pull request merged**